tests.txt

```diff
@@ -11,14 +11,26 @@ Eggs are placed in our freshly created eggs directory:
     >>> # Use a specific grok version, because 'current' will change at some
     >>> # point in the future.
     >>> executable = [os.path.join(current_dir, 'bin', 'grokproject')]
+
+When we have merged the PR, this will need to change
+
+    >>> base = open(os.path.join(current_dir, '.git', 'config')).read()
+    >>> base = base[base.find('url = ')+6:base.find('.git')]
+    >>> base = base.replace('github.com', 'raw.githubusercontent.com')
+
+We temporarily hard code 'prsephton/grokproject' since the version info does not
```

Comment:
Do you mean you'll remove the hardcoded paths after this PR is merged? And will that removal then be in its own PR?

Reply:
Yes.  The first PR will be using the 'github/prsephton/grokproject' repo for a 'versions' directory.  Once merged we should have a 'versions' dir in zopefoundation/grokproject, and the second PR will fix the code to use that. 

The rain dance is just to ensure that things are never in a broken state and that the code can always reach a working 'versions' directory.